lib/core.ps1

```diff
@@ -838,6 +892,13 @@ function shim($path, $global, $name, $arg) {
         if ($arg) {
             Write-Output "args = $arg" | Out-UTF8File "$shim.shim" -Append
         }
+
+        $target_subsystem = Get-Subsystem $resolved_path
+
+        if (($target_subsystem -ne 3) -and ($target_subsystem -ge 0)) { # Subsystem -eq 3 means `Console`, -ge 0 to ignore
+            Write-Output "Changing Subsystem 3[current] to $target_subsystem[new] for $shim.exe"
```

Comment:
Could we change this message to something more user-friendly? I'm sure most of the users won't know what a PE header or a subsystem is - they might get scared off thinking this is an error.